CONTRIBUTING.md

```diff
@@ -37,6 +37,7 @@ After the dependencies were installed and the VM initialized, you can proceed wo
 # Pull Request Standards 
  - Each Pull Request should focus on single responsibility principle.
  - **The Pull Request should not break any of the existing functionality**
+ - PRs title: <Component name>:  explanation in a few words
```

Comment:
Is it the right place for changing the CONTRIBUTING.md file?

Reply:
what do you mean? you have a different place in mind?